### PR TITLE
fix: update addon-resizer and cluster-proportional-autoscaler to fix CVEs

### PR DESCRIPTION
**Pull request by ganeshkumar5699**
**What type of PR is this?**
/kind update

**What this PR does / why we need it**:
These components are using an older version with CVEs, that has been flagged. Updating to fix that.

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -2,11 +2,9 @@
   "ContainerImages": [
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/autoscaler/addon-resizer:*",
-      "amd64OnlyVersions": [
-        "1.8.5"
-      ],
+      "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "1.8.14"
+        "1.8.18"
```
Comment (alexeldeib) [inline vhdbuilder/packer/components.json:7]: should we just delete these? do any addons use them?

**Review by alexeldeib — APPROVED** [review]
lgtm but deleting maybe better?

File: vhdbuilder/packer/components.json (hunk 1 above)
Comment (ganeshkumar5699) [reply vhdbuilder/packer/components.json:7]: These components are being referenced by addons (though different versions), so keeping the components for now.

**Comment by ganeshkumar5699** [comment]
> lgtm but deleting maybe better?

Want to delete if possible too. Components do have references by certain addons, so keeping them for now.

**Pull request merged**